docs/maui/views/Popup.md

```diff
@@ -103,43 +103,44 @@ There are 2 different ways that a `Popup` can be closed; programmatically or by
 
 ### Programmatically closing a Popup
 
-In order to close a `Popup` a developer must call `Close` or `CloseAsync` on the `Popup` itself. This is typically performed by responding to a button press from a user.
+In order to close a `Popup` a developer must call `Close` or `CloseAsync` on the `Popup` itself. This is typically performed by responding to a button press from a user or once a long running task has completed.
 
-We can enhance the previous XAML example by adding an **OK** `Button`:
+To show how a `Popup` can be closed programmatically, the `SimplePopup` XAML example can be enhanced by:
 
-```xml
-<toolkit:Popup xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
-               xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-               xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
-               x:Class="MyProject.SimplePopup">
+- changing the `ContentView` element to `VerticalStackLayout`
+- adding an **OK** `Button`
 
-    <VerticalStackLayout>
-        <Label Text="This is a very important message!" />
-        <Button Text="OK" 
-                Clicked="OnOKButtonClicked" />
-    </VerticalStackLayout>
+```xaml
+<VerticalStackLayout
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="MyProject.SimplePopup">
+
+    <Label Text="This is a very important message!" />
+    <Button Text="OK" 
+            Clicked="OnOKButtonClicked" />
     
-</toolkit:Popup>
+</VerticalStackLayout>
 ```
 
-In the resulting event handler we call `Close`, this will programmatically close the `Popup`.
-
-> [!NOTE]
-> `Close()` is a fire-and-forget method. It will complete and return to the calling thread before the operating system has dismissed the `Popup` from the screen. If you need to pause the execution of your code until the operating system has dismissed the `Popup` from the screen, use instead `CloseAsync()`.
+In the resulting event handler `OnOKButtonClicked` we call `Close`, this will programmatically close the `Popup`.
 
 ```csharp
-public partial class MySimplePopup : Popup
+public partial class SimplePopup : VerticalStackLayout
```

Comment:
Before `SimplePopup` was a `ContentView` here it's a `VerticalStackLayout`, is that intentional? If so, wouldn't be good to describe this change?

Reply:
This is explained on line 110 but perhaps that isn't clear enough?

Reply:
Oh so I missed that, ex just reading the diffs